src/cli/benchmark.py

```
@@ -982,6 +1012,16 @@ def run(
         "--compare",
         help="Path to baseline JSON file for regression comparison.",
     ),
+    transcribe_smoke: bool = typer.Option(
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)